_posts/2016-07-07-Rust-1.10.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: post
+title: "Announcing Rust 1.10"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.10. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.10][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.10][notes] on GitHub.
+1276 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1100-2016-07-07
+
+### What's in 1.10 stable
+
+Rust 1.10 contains one of the most-requested features in Rust: through the [`-C
+panic=abort` flag] or a [setting in your `Cargo.toml`], when a `panic!` happens,
+it will abort rather than unwind. Why would you want to do this? With an
+abort, less code gets generated, meaning that binary sizes are a bit smaller,
+and compilation time is ever-so-slightly faster. You give up the ability to
+recover, however.
+
+[`-C panic=abort` flag]: https://github.com/rust-lang/rust/pull/32900
+[setting in your `Cargo.toml`]: https://github.com/rust-lang/cargo/pull/2687
+[RFC 1513]: https://github.com/rust-lang/rfcs/blob/master/text/1513-less-unwinding.md
+
+The second big feature in 1.10 is a new crate type: [`cdylib`]. Rust already has
```

Comment:
I think it might be more clear to say:

> The second big feature in 1.10 is a new crate type: [`cdylib`]. The existing `dylib` dynamic library format will now be used solely for writing a dynamic library to be used within a Rust project, while `cdylib`s will be used when compiling Rust code as a dynamic library to be embedded in another language. With the 1.10 release, `cdylib`s are supported by the compiler, but not yet in Cargo.

Also, it doesn't look like the RFC is actually linked to?